### DISCUSSION TITLE
Disable emojis

### DISCUSSION THREAD

**Discussion by y00ni** [community/community#3889] [Discussions]
Hello,

I love the use of emojis as a reaction to all posts/comments in the discussion. But it'd also be great to DISABLE emojis so that moderators can choose when users cannot react through emojis on anything. 

Thanks!

**Comment by sudo-nano** [comment]
I would love this, and I would love to be able to disable the emojis associated with discussion categories. They're obnoxious.

**Comment by Bubrak-Sher** [comment]
Proposed Solution:

A toggle in repository discussion settings to enable/disable emoji reactions.

Optional override at the discussion level.

Possibly scoped by discussion category or individual roles (e.g., allow reactions only from maintainers).
